tests/Spec.hs

```diff
@@ -62,8 +62,38 @@ main =
     , integralSpec @CIntMax
     , integralSpec @CUIntMax
     , integralSpec @Integer
+    , rangeSpec @Word8
```

Comment:
There is no need for all of these `rangeSpec` calls here, `integralSpec` invokes `rangeSpec`

I structured it in such a way that you only need specify  a type only once and all of the applicable specs should go in there, precisely to avoid this sort of redundancy

Reply:
My bad! I've removed the `rangeSpec` calls for `Char` and `Bool` now since they are already covered by `integralSpec`: https://github.com/idontgetoutmuch/random/pull/58/commits/feac91f231fc4d0b7943475c248f8fe273423deb#diff-47e9594b3c7ba1c5f6e32c4a96aead95R43-R44